doc/project/checklist.md

```diff
@@ -82,9 +82,9 @@ Feature requests for this IP version are frozen at this time.
 
 All features specified have been completed.
 
-### AREA_SANITY_CHECK
+### AREA_CHECK
 
-Area sanity check completed either on FPGA or on Design Compiler.
+Area check completed either on FPGA or on Design Compiler.
```

Comment:
@tjaychen @eunchan 

Should we replace `Design Compiler` with something more generic?

E.g., `Area check completed either in FPGA or ASIC technology.`

Reply:
hmm good question.
I feel like it's okay to say design compiler. I don't think there's necessarily any reason to hide what tool we are using.
Of course, if this is ever enhanced to support multiple tools, then it definitely would make sense to update that.